google-api-go-generator/testdata/json-body.json

```diff
@@ -451,7 +451,7 @@
                   "type": "string"
                 },
                 "parent": {
-                  "description": "Required. The name of the project for which available locations are to be\nlisted (since some locations might be whitelisted for specific projects).",
+                  "description": "Required. The name of the project for which available locations are to be\nlisted (since some locations might be allowlisted for specific projects).",
```

Comment:
Perhaps "restricted to"? It's not clear what this means (projects can only use certain regions, or regions can only be used by certain projects?)

Reply:
Ack.